daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Node.scala

```diff
@@ -422,3 +422,13 @@ object Node {
     type WithTxValue[+Nid, +Cid] = F[Nid, Cid, Transaction.Value[Cid]]
   }
 }
+
+/** The constructor is private so that we make sure that only this object constructs
+  * node ids -- we don't want external code to manipulate them.
+  */
```

Comment:
This was simply copied and pasted here but I agree with Miklos' comment, this comment can simply go away.
```suggestion
```

Reply:
:+1: